spec/function.dd

```diff
@@ -2884,10 +2884,60 @@ $(H3 $(LNAME2 safe-functions, Safe Functions))
         so they can be corrected.
         )
 
+$(H4 Safe External Functions)
+
+        $(P External functions don't have a function body visible to the compiler:
+        ---
+        @safe extern (C) void play();
+        ---
+        and so safety cannot be verified automatically.
+        )
+
+        $(BEST_PRACTICE Explicitly set an attribute for external functions rather
+        than relying on default settings.)
+
 $(H3 $(LNAME2 trusted-functions, Trusted Functions))
 
         $(P Trusted functions are marked with the $(CODE @trusted) attribute.)
 
+        $(P Trusted functions act as an interface between safe and system functions.
+        A trusted function provides a safe interface. A $(I Safe Interface) is one that:
+
+        $(UL
```

Comment:
Same thing as with the code block above: Do not put `$(UL ...)` into `$(P ...)`. HTML doesn't allow it. The empty third paragraph in the output is because of this.

Reply:
ok